web/templates/web/header.html

```diff
@@ -72,9 +74,9 @@
         </a>
     </nav>
 
-    <nav id="side-nav" class="ui secondary huge inverted menu">
+    <div id="side-nav" class="ui secondary huge inverted menu">
```

Comment:
Functionally (for most users), it makes not difference, but it can make a difference for users with screen readers, as it adds a `role="navigation"` attribute (see [ARIA: Navigation Role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#Description)). Additionally, there was another `nav` element nested inside this one - which caused there to be two of those `"navigation"` roles for the same set of navigation links, and the inner one contained the links the most "directly", so I thought this made the most sense 🙂 See [the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) for more info.